crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E274_E27.py.snap

```diff
@@ -2,17 +2,73 @@
 source: crates/ruff/src/rules/pycodestyle/mod.rs
 expression: diagnostics
 ---
+- kind:
+    name: TabBeforeKeyword
```

Comment:
Confused why this one only triggers once, but row 12 triggers twice. In both cases we have two tabs.